src/Core/src/Platform/Android/MauiScrollView.cs

```diff
@@ -214,7 +214,7 @@ protected override void OnLayout(bool changed, int left, int top, int right, int
 		{
 			base.OnLayout(changed, left, top, right, bottom);
 
-			if (_hScrollView?.Parent == this && _content is not null)
+			if (changed && _hScrollView?.Parent == this && _content is not null)
```

Comment:
The issue still happens after merge https://github.com/dotnet/maui/pull/22347?

Reply:
PR22347  has fixed this issue and performed very well.
This PR attempts to solve the problem from another perspective, that is, the _hScrollView should not be adjusted when 'changed' is' false '.